DOCKER.md

```diff
@@ -101,90 +92,69 @@ These tests:
 
 ## Configuration with MCP Clients
 
-To use the Docker container with MCP clients (such as Claude or other AI models), add the server configuration to your client's settings:
+To use the Docker container with MCP clients (such as Claude or other AI models), add the server configuration to your client's settings.
 
-### Claude/Anthropic MCP Configuration
+### MCP Client Configuration
 
-In the `.mcpconfig.json` file (or corresponding environment variables):
+You can get the recommended configuration by running:
 
-```json
-{
-  "mcpServers": {
-    "troubleshoot": {
-      "type": "stdio",
-      "command": "/path/to/scripts/run.sh",
-      "args": [
-        "--mcp"
-      ],
-      "env": {
-        "SBCTL_TOKEN": "${SBCTL_TOKEN}"
-      }
-    }
-  }
-}
+```bash
+docker run --rm mcp-server-troubleshoot:latest --show-config
 ```
 
-Or if you prefer using docker directly:
+The output will provide a ready-to-use configuration for MCP clients:
 
 ```json
 {
   "mcpServers": {
     "troubleshoot": {
-      "type": "stdio",
       "command": "docker",
       "args": [
         "run",
         "-i",
         "--rm",
         "-v", 
-        "${LOCAL_BUNDLE_DIRECTORY}:/data/bundles",
+        "${HOME}/bundles:/data/bundles",
         "-e",
         "SBCTL_TOKEN=${SBCTL_TOKEN}",
         "-e",
         "MCP_BUNDLE_STORAGE=/data/bundles",
```

Comment:
Again this is a default and shouldn't be needed to be set or included in recommended settings.